examples/amp-subscriptions.amp.html

```diff
@@ -329,10 +329,10 @@ <h1 itemprop="headline">Lorem Ipsum</h1>
   </footer>
 
 
-  <template type="amp-mustache" subscriptions-dialog subscriptions-display="value = 'B'">
-    This is dialog (B) from template!
+  <template type="amp-mustache" subscriptions-dialog subscriptions-display="entitlement.service = 'local'">
```

Comment:
Would rather not advertise the expressions of the type `enttilement.service = local`. B/c these sections are not rendered at all when it's not local. And when we decide to support multiple local service, I'd rather be more explicit with `susbcriptions-service="name"` attribute. This is b/c I want to always know which section belongs to who. But for now they all belong to the default "local".